node/npm/glide/index.ts

```diff
@@ -138,7 +140,6 @@ function initialize() {
         FtAggregateApply,
         FtAggregateReturnType,
         FtSearchReturnType,
-        FtProfileOtions,
```

Comment:
Why removing this and others classes/options?

Reply:
`FtProfileOtions` doesn't exist... it's an unnecessary export.